src/components/Loading.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from '../styles/loading'
+
+import React from 'react'
+
+const Loading = ({ message }) => {
+  return (
+    <div className={styles['fil-loading']}>
+      <p>{message}</p>
+    </div>
+  )
+}
+
+export default Loading
```

Comment:
It should, as it's used in all our apps now. But I also think this PR is big enough as it is, so maybe that can be done separately.